### PR TITLE
Fixes for issue 895 (and additional error found)

### PR DESCRIPTION
**Pull request by davidpesce**
Fixes for #895. First commit had IDE formatting issues. Full PR should resolve that.

Previous code returned a success message regardless of the EloquentUserRepository.php verifyEmail function. This should be resolved now.

### CODE REVIEW

File: app/locker/repository/User/EloquentUserRepository.php [hunk 1]
```diff
@@ -51,21 +51,16 @@ public function verifyEmail($token){
 
     //first see if a record exists for that email and token
     $email = \DB::table('user_tokens')
```
Comment (ryansmith94) [inline app/locker/repository/User/EloquentUserRepository.php]: This looks much better! Thanks @davidpesce 👍

**Comment by ryansmith94** [comment]
Hi David, I still get the issue unfortunately.

Screen recording

**Comment by davidpesce** [comment]
It's been a bit since I've used Laravel, but I thought I traced that through correctly. Let me double-check.

**Comment by ryansmith94** [comment]
No worries @davidpesce

**Comment by davidpesce** [comment]
Here's the only other place I can find a reference to reminders.reset. https://github.com/LearningLocker/learninglocker/blob/develop/app/views/system/forms/login.blade.php#L8

If any message contains 'success' it'll display the password reset message. I wouldn't expect to see logic like that in a view. Thoughts?

**Comment by davidpesce** [comment]
It looks like it is in others. I'll follow suit and attempt a fix.

**Comment by ryansmith94** [comment]
Thanks @davidpesce, I'll give this another test. Bit busy at the moment though unfortunately, so I've setup a reminder for tomorrow morning.

**Comment by ryansmith94** [comment]
Ok just tested this @davidpesce, all working now, nice work! Just going to review code before I merge.

**Comment by ryansmith94** [comment]
All done thanks @davidpesce! 👍 This fix will go into the next release, we'll probably make that release next Friday. In the meantime, the fix is available via the `develop` branch.

**Pull request merged**